tests/pos/i16236.scala

```diff
@@ -0,0 +1,9 @@
+trait A
```

Comment:
Could you make a copy of this test and copy to `tests/explicit-nulls/pos` as well? Thanks